source/crud/read-operations/skip.txt

```diff
@@ -0,0 +1,58 @@
+=====================
+Skip Returned Results
+=====================
+
+.. default-domain:: mongodb
+
+
+Use skip to omit documents from the beginning of the list of returned
+documents for a read operation. You can combine skip with sort to omit
+the top (for descending order) or bottom (for ascending order) ``N``
+results for a given query. Since the order of documents returned is
+:manual:`not guaranteed
+</reference/method/cursor.sort/#result-ordering>` in the absence of a
+sort, using skip without using sort omits arbitrary documents.
+
+If the value of skip exceeds the number of matched documents for your
+query, your query will return no documents.
+
+Consider a collection containing documents that describe varieties of
```

Comment:
You might consider adding a transition sentence here that specifically signals to the reader that they are about to start a multi-step example. One idea is "Follow the instructions in the examples below to insert data in a collection and perform a sort and skip on the query results.

Reply:
Similar comment for the `read-operations/sort.txt file`